wire-runtime/src/test/proto/one_of.proto

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package squareup.protos.oneof;
+
+option java_package = "com.squareup.wire.protos.oneof";
+
+message OneOfMessage {
+
+  // Must have a foo or a bar.
+  oneof choice {
+    // What foo.
+    int32 foo = 1;
+    // Such bar.
+    string bar = 2;
```

Comment:
Can you either flip the tag values or set this to 3 so that we ensure we're testing that tag values are used in the choice enum?